experiments/benchmarks/bucket.jl

```diff
@@ -141,9 +141,7 @@ function setup_prob(t0, tf, Δt; nelements = (100, 10))
     return prob, cb
 end
 
-function setup_and_solve_problem(; greet = false)
-    # We profile the setup phase as well here. This is not intended, but it is the easiest
-    # to set up for both CPU/GPU at the same time
+function setup_one(; greet = false)
```

Comment:
why this function name?

Reply:
`one`, as one instance of a problem that is fully ready to be solved

Reply:
perhaps "setup_simulation" is clearer, or "setup_instance"? as you prefer.